docs/README.md

```diff
@@ -30,19 +29,18 @@ If nebula.gl is more than what you need (e.g. in terms of bundle size), and you
 
 That said, if you are already using `deck.gl` the additional overhead of nebula.gl is small, and the seamless integration with deck.gl should be valuable.
 
-
```

Comment:
This section is now clearly outdated and simply wrong! We should rephrase this section to something like:

nebula.gl provides multiple npm packages: 

- A fully featured deck.gl integration: ultra-performant, fully 3D-enabled editing UI, packaged as a suite of editable deck.gl layers.
- A minimal React integration: Light weight SVG based editing UI, primarily for react-map-gl applications that do not want the full deck.gl dependency.
- A UI independent Editing engine that can be integrated into any application